sources/packages/web/src/views/institution/student/InstitutionStudentApplicationView.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <full-page-container v-if="applicationDetail.data">
+    <template #header>
+      <header-navigator
+        title="Back to student applications"
+        :routeLocation="{
+          name: InstitutionRoutesConst.STUDENT_APPLICATIONS,
+          params: { studentId },
+        }"
+        subTitle="Financial Aid Application"
+      >
+      </header-navigator>
+    </template>
+    <h2 class="color-blue pb-4">
+      Student Application Details
+      {{
+        applicationDetail.applicationNumber
+          ? HYPHEN_WITH_SPACE + applicationDetail.applicationNumber
```

Comment:
https://github.com/bcgov/SIMS/pull/1914/files#r1184384600

Reply:
https://github.com/bcgov/SIMS/pull/1914#discussion_r1184417241

Reply:
removed it and used emptyStringFiller